pkgs/development/interpreters/php/default.nix

```diff
@@ -195,9 +195,10 @@ let
 
           hardeningDisable = [ "bindnow" ];
 
-          preConfigure = ''
-            # Don't record the configure flags since this causes unnecessary
-            # runtime dependencies
+          preConfigure =
```

Comment:
It would probably have been better to skip the whitespace changes here, since they will cause rebuilds for pho >= 7.4. ofBorg says its in the <=500 range though, which should be okay.

Reply:
True, as this is a rush fix.

I should put in some docs somewhere that comments outside of string literals is preferred for sake of rebuilds, so as I am not tempted to do this on a case-by-case basis so much :).